test/cmdlineTests.sh

```diff
@@ -103,12 +103,19 @@ printTask "Compiling various other contracts and libraries..."
 cd "$REPO_ROOT"/test/compilationTests/
 for dir in *
 do
-    if [ "$dir" != "README.md" ]
+    ignoreDir=false
+    if [ "$dir" == "corion" ] && [ "$CIRCLECI" ] && [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
What message is this failing with? I doesn't fail on my setup.

Reply:
@axis ``CompilerStack::compileContract()`` fails with 
```
Dynamic exception type: boost::exception_detail::clone_impl<dev::solidity::InternalCompilerError>
std::exception::what: Assembly exception for bytecode
[dev::tag_comment*] = Assembly exception for bytecode
```
Looking into it right now...

Reply:
Me neither...Started by commenting out stuff in the contracts itself in order to isolate files, but haven't found the issue yet.

Reply:
Just remove the `catch` statement and you'll get the underlying exception.

Reply:
This might be cause by an assembly tag not found and the root cause might be the size_t thing we just fixed - hopefully.

Reply:
This: https://github.com/ethereum/solidity/blob/develop/libsolidity/interface/CompilerStack.cpp#L730

Reply:
It works!